gene2phenotype_project/gene2phenotype_app/serializers/locus_genotype_disease.py

```diff
@@ -512,10 +550,12 @@ def update(self, instance, validated_data):
     def update_mechanism(self, lgd_instance, validated_data):
         """
             Method to update the molecular mechanism of the LGD record.
-            It only allows to update mechanisms with value 'undetermined'
-            and support value 'inferred'.
+            It only allows to update the mechanism value if current value is 'undetermined'.
 
             Mandatory fields are molecular_mechanism name and support (inferred/evidence).
+            If you don't want to update the mechanism value then input an empty string: "name": ""
+            Same for the support: "support": ""
+
```

Comment:
can the api return 400 bad request if the mechanism value anything other than "undetermined" or ""

Reply:
"undetermined"?